manifests/0000_30_cluster-api_capi-operator_01_images.configmap.yaml

```
@@ -12,13 +12,8 @@ data:
   images.json: >
     {
       "cluster-capi-operator": "registry.ci.openshift.org/openshift:cluster-capi-operator",
-      "cluster-api:operator": "quay.io/asalkeld/cluster-api-operator-amd64:dev",
-      "bootstrap-kubeadm:manager": "k8s.gcr.io/cluster-api/kubeadm-bootstrap-controller:v0.4.3",
-      "controlplane-kubeadm:manager": "k8s.gcr.io/cluster-api/kubeadm-control-plane-controller:v0.4.3",
-      "core-cluster-api:manager": "k8s.gcr.io/cluster-api/cluster-api-controller:v1.0.1",
-      "infrastructure-aws:manager": "k8s.gcr.io/cluster-api-aws/cluster-api-aws-controller:v0.7.0",
-      "infrastructure-azure:manager": "us.gcr.io/k8s-artifacts-prod/cluster-api-azure/cluster-api-azure-controller:v0.5.2",
-      "infrastructure-metal3:ip-address-manager": "quay.io/metal3-io/ip-address-manager:v0.1.0",
-      "infrastructure-metal3:manager": "quay.io/metal3-io/cluster-api-provider-metal3:v0.5.0",
+      "cluster-kube-cluster-api-operator": "quay.io/ademicev/cluster-api-operator:latest",
+      "cluster-capi-controllers": "registry.ci.openshift.org/openshift:cluster-capi-controllers",
+      "aws-cluster-api-controllers": "quay.io/ademicev/cluster-api-provider-aws:latest",
```

Even though it doesn't exist yet, why not do this so that it matches what it should be eventually?
```suggestion
      "aws-cluster-api-controllers": "registry.ci.openshift.org/openshift:cluster-api-provider-aws",
```

---

This is for development only, there is no downstream image for aws now, having `quay.io` allows us to test the changes using CI. I will replace it once ART requests are fulfilled.